modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/DynamicMetadataDeclarationEnricher.java

```diff
@@ -155,14 +189,15 @@ private void declareResolversInformation(ExecutableComponentDeclaration<? extend
         String attributesResolver = metadataScope.getAttributesResolver().get().getResolverName();
         String keysResolver = metadataScope.getKeysResolver().get().getResolverName();
 
-        //TODO MULE-15638 - Once Metadata API 2.0 is implemented we will know better if the resolver requires or not a connection of config.
+        // TODO MULE-15638 - Once Metadata API 2.0 is implemented we will know better if the resolver requires or not a connection
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7668&issues=AWjdfEM0otIXnmsUGfbZ&open=AWjdfEM0otIXnmsUGfbZ)

[ISSUE_KEY]: AWjdfEM0otIXnmsUGfbZ